File: 1784.检查二进制字符串字段.js
/*
 * @lc app=leetcode.cn id=1784 lang=javascript
 *
 * [1784] 检查二进制字符串字段
 */

// @lc code=start
/**
 * @param {string} s
 * @return {boolean}
 */
var checkOnesSegment = function(s) {
    return s.indexOf('01') === -1;
};
// @lc code=end

File: 1784.检查二进制字符串字段.java
/*
 * @lc app=leetcode.cn id=1784 lang=java
 *
 * [1784] 检查二进制字符串字段
 */

// @lc code=start
class Solution {
    public boolean checkOnesSegment(String s) {
        // 出现 01 就不符合,其余情况都符合
        return !s.contains("01");
    }
}
// @lc code=end

